player/models.py

```diff
@@ -130,6 +130,11 @@ def push(self, music_id, requestId=None, **kwargs):
             self.current_music = music
             self.save()
             self.play_next(forced=True)
+        else:
```

Comment:
This piece of code is really confuse and not documented...
the push function just push music in the queue (and start the music if the queue is empty)
I should refactor this (it's easy)

Your code should be in the play function